BLECarPiZeroW/BLEHandler.py

```diff
@@ -62,6 +85,7 @@ class BLEAdvertisement(Advertisement):
     def __init__(self, bus, index):
         Advertisement.__init__(self, bus, index, 'peripheral')
         self.add_service_uuid(MovementService.MVMT_SVC_UUID)
+        self.add_service_uuid(IpDiscoveryService.IPDSC_SVC_UUID)
         self.include_tx_power = True
 
```

Comment:
Add a new advertisement for the IP Service here, to be registered in main.
```
class IPAdvertisement(Advertisement):
    def __init__(self, bus, index):
        Advertisement.__init__(self, bus, index, 'peripheral')
        self.add_service_uuid(IpDiscoveryService.IPDSC_SVC_UUID)
        self.include_tx_power = True
```
Inside main, add:
```
ip_advertisement = IPAdvertisement(bus,1)
[...]
# Register advertisement for IP Service
    ad_manager.RegisterAdvertisement(ip_advertisement.get_path(), {},
                                     reply_handler=register_ad_cb,
                                     error_handler=register_ad_error_cb)
```